doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -286,6 +286,7 @@ nRF9160 samples
 
     * CoAP max message size is set to 1280 by default.
     * Number of SenML CBOR records is set to a higher value to cope with data exchange after registration with Coiote server.
+    * Sample has been switched to use TF-M instead of SPM.
```

Comment:
```suggestion
* Updated:

    * Sample now uses TF-M instead of SPM.
```

Reply:
I went already with Joakim's proposal. Hopefully that is fine for you.

Reply:
We use 'Added', 'Updated' and 'Fixed' as the main labels throughout the release notes/changelog. Suggesting to make it consistent.

Reply:
@umapraseeda I suggested "changed" because there is one further down. So you might want to update/change that one.

Reply:
I shall fix that in another PR with some general fixes.